### ISSUE TITLE
Think about how to allow arguments to `display_width`

### ISSUE THREAD

**Issue by zhiburt** [zhiburt/tabled#159]
I was thinking about this example recently.

https://github.com/zhiburt/tabled/blob/a5833e807e064a09d6208835874377fd709ca514/examples/nested_table_2.rs#L7-L17

We actually could hide the display_distribution method by a little function like this.

```rust
pub fn display_table<T>(t: &T) -> String
where
    T: Tabled,
{
    Table::new([t]).to_string()
}
```

But this function will have only constant style.
We can't tweak it using `display_width`.

I was thinking to add a new argument like this.

```
#[tabled(display = "table")]

#[tabled(display("table", formatting = ""))]
```

But it's clearly not aligned with `Tabled` trait. And it should not be.

**Comment by zhiburt** [comment]
ref: #126 #158

**Comment by zhiburt** [comment]
I think it closed at least partially by #169
